src/components/Calendar/Calendar.tsx

```diff
@@ -71,6 +71,17 @@ const CalendarStyling = styled.div`
     }
 `;
 
+const endDateInclusiveEvents = (endDateExclusiveEvents: CalendarEvent[]): CalendarEvent[] => {
+    return endDateExclusiveEvents.map((exclusiveEvent: CalendarEvent): CalendarEvent => {
+        const cloneEndDate = new Date(exclusiveEvent.end);
```

Comment:
I think 'copied' is better than 'cloned' but this is just a 'use simple language' thing

Reply:
Refactored as suggested